normalizer.py

```diff
@@ -0,0 +1,59 @@
+# Lint as: python3
+"""Normalizes text by applying the text normalizer.
+
+Either normalizes the text from a flag, or loads an external file of
+sentences to normalize. If it uses the external file, it will write
+the sentences that were changed to a new file.
+"""
+
+from typing import List
+from tqdm import tqdm
+from absl import app
+from absl import flags
+import normalizer_lib
+import preprocess
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('string_to_normalize', None, 'the string to normalize')
+
+UD_FILE: str = "./language_data/af/UD_Afrikaans-AfriBooms/af_afribooms-ud-train.conllu"
```

Comment:
Can we rename these to not be "Universal Dependencies" specific?  Maybe something like "DATA_PATH" and "DATA_TEXT".

Reply:
We also want to keep this language netural here, so this should perhaps be a flag that takes a command line argument filepath

Reply:
That makes sense. One complication is that right now it's calling the `process_ud_data` function from `preprocess.py`, which depends on the format of the UD data but not the other data sources. This should be easily fixed by making a generic `preprocess` function called here and adding different cases in `preprocess.py` that will vary depending on the data source.

Since this PR doesn't include `preprocess.py`, I'll change it here in `normalizer.py` for now and make sure to add those changes to the other file.